web/src/components/Stats.svelte

```diff
@@ -13,11 +13,22 @@
     class="flex bg-green-700 xs:space-x-10 justify-around xs:justify-center items-center p-2 2xs:p-4 shadow-lg"
 >
     {#each stats as stat}
-        <div class="flex justify-between items-center flex-col">
+        <div
+            class="flex justify-between items-center flex-col"
+            title={stat.name}
+        >
             <div
                 class="w-8 h-8 2xs:w-10 2xs:h-10 xs:w-12 xs:h-12 rounded-full bg-white grid place-items-center shadow-lg mb-2 xs:mb-3 text-black text-sm"
             >
-                {stat.name.slice(0, 2)}
+                <div class="w-8 h-8 grid place-items-center">
+                    <img
```

Comment:
I think we should use `object-contain` here in order to make sure this works for non-square icons.